### DISCUSSION TITLE
reload or rerender the Editor in react project

### DISCUSSION THREAD

**Discussion by pswu11** [GrapesJS/grapesjs#5492] [Q&A]
Hi, I'm quite new to the project and I really like it!

I've implemented GrapeJS in a React project where I have several web projects per user with dynamic routing such as`url/projects/:projectId`.

For each project, I saved the project to localStorage as `gjs-${projectId}` at the moment, but I'm facing a problem - when user navigates to different routes (switching to another project), the `<GjsEditor />` won't reload the the design from respective localStorage unless I reload the page. 

Is there anyway to trigger the reload of this component (including storageManager) without reloading the whole page?
Thank you in advance!

**Comment by artf** [comment]
Hi @pswu11 I guess this is a matter of understanding how to tell your React application to rerender the editor on route change, not really related to GrapesJS itself.
If you're able to provide at least a reproducible demo, maybe someone will be able to help you.

**Reply by pswu11** [reply]
@artf thank you! yes i think it's about react and I'm rather new to react but for now I managed to make it work without using grapejs' react wrapper. 🙏🏼
